cluster/node-pools/master-default/userdata.clc.yaml

```diff
@@ -558,6 +558,36 @@ storage:
             volumeMounts:
             - name: config-volume
               mountPath: /etc/nginx
+          - name: skipper-proxy
+            image: registry.opensource.zalan.do/pathfinder/skipper:v0.10.233
+            args:
+            - skipper
+            - -address=:8443
+            - -tls-cert=/etc/kubernetes/ssl/apiserver.pem
+            - -tls-key=/etc/kubernetes/ssl/apiserver-key.pem
+            - -insecure
+            - -inline-routes
+            - 'z: JWTPayloadAnyKV("iss", "https://identity.zalando.com") -> disableAccessLog() -> "https://127.0.0.1:443"; all: * -> enableAccessLog() -> "https://127.0.0.1:443";'
```

Comment:
We might need `-experimental-upgrade` for `exec` to work. @szuecs WDYT?

Reply:
yes

Reply:
@szuecs thanks, would you recommend any other flags?

Reply:
-runtime-metrics
-enable-connection-metrics
-enable-prometheus-metrics

You should also test if port-forwarding and exec works as expected and how long the connection is working. Maybe we should adjust some timeouts and idle connection handling.

Reply:
added